### ISSUE TITLE
I am getting ideviceinfo returned an error:                 ERROR: Could not connect to lockdownd, error code -8

### ISSUE THREAD

**Issue by MuthuHere** [flutter/flutter#47727]
![image](https://user-images.githubusercontent.com/18752533/71405338-1d6b8600-2670-11ea-95df-d57edc35d6b1.png)

**Comment by iapicca** [comment]
Hi @MuthuHere 
could you please run 
`flutter channel stable`
`flutter upgrade --force`
`flutter clean`
`flutter pub cache repair`
If the issue persist please provide your updated 
`flutter doctor -v` and `flutter run --verbose`
together with a a minimal code sample to reproduce the problem.

Please refrain from posting pictures if not necessary.

thank you

**Comment by sadohert** [comment]
tl;dr - I was seeing this error when I tried to run my flutter app on the simulator in VS Code.  I unplugged an old iPod Touch (4th Gen) and rebooted my computer and the issue went away.

#### More Info
I was getting the error:
```
flutter ideviceinfo returned an error: ERROR: Could not connect to lockdownd, error code -17
```

Running `ideviceinfo -d` produced similar errors on its own too.

I hadn't seen any updates to flutter or plugins and was working just fine this morning.  The only thing I had done different was to plug an old iPod Touch 4th Gen into laptop.

Tried the steps above and it didn't resolve the issue for me.

`flutter doctor -v`:

```
[✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Mac OS X 10.15.2 19C57,
    locale en-CA)
    • Flutter version 1.12.13+hotfix.5 at /Users/stu/personal_dev/flutter
    • Framework revision 27321ebbad (2 weeks ago), 2019-12-10 18:15:01 -0800
    • Engine revision 2994f7e1e6
    • Dart version 2.7.0


[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/stu/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/stu/Library/Android/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.7.5

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 39.0.3
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.41.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.7.1

[☠] Connected device (the doctor check crashed)
    ✗ Due to an error, the doctor check did not complete. If the error message
      below is not helpful, please let us know about this issue at
      https://github.com/flutter/flutter/issues.
    ✗ Exception: ideviceinfo returned an error:
      ERROR: Could not connect to lockdownd, error code -17

    • #0      IMobileDevice.getInfoForDevice
      (package:flutter_tools/src/ios/mac.dart:226:9)
      <asynchronous suspension>
      #1      IOSDevice.getAttachedDevices
      (package:flutter_tools/src/ios/devices.dart:190:55)
      <asynchronous suspension>
      #2      IOSDevices.pollingGetDevices
      (package:flutter_tools/src/ios/devices.dart:115:57)
      #3      PollingDeviceDiscovery.devices
      (package:flutter_tools/src/device.dart:285:52)
      #4      DeviceManager.getAllConnectedDevices
      (package:flutter_tools/src/device.dart:140:46)
      <asynchronous suspension>
      #5      DeviceValidator.validate
      (package:flutter_tools/src/doctor.dart:851:54)
      #6      asyncGuard.<anonymous closure>
      (package:flutter_tools/src/base/async_guard.dart:111:32)
      #7      _rootRun (dart:async/zone.dart:1126:13)
      #8      _CustomZone.run (dart:async/zone.dart:1023:19)
      #9      _runZoned (dart:async/zone.dart:1518:10)
      #10     runZoned (dart:async/zone.dart:1502:12)
      #11     asyncGuard (package:flutter_tools/src/base/async_guard.dart:109:3)
      #12     Doctor.startValidatorTasks
      (package:flutter_tools/src/doctor.dart:158:9)
      #13     Doctor.diagnose (package:flutter_tools/src/doctor.dart:252:41)
      #14     _AsyncAwaitCompleter.start
      (dart:async-patch/async_patch.dart:45:6)
      #15     Doctor.diagnose (package:flutter_tools/src/doctor.dart:241:24)
      #16     DoctorCommand.runCommand
      (package:flutter_tools/src/commands/doctor.dart:59:39)
      #17     _AsyncAwaitCompleter.start
      (dart:async-patch/async_patch.dart:45:6)
      #18     DoctorCommand.runCommand
      (package:flutter_tools/src/commands/doctor.dart:45:42)
      #19     FlutterCommand.verifyThenRunCommand
      (package:flutter_tools/src/runner/flutter_command.dart:615:18)
      #20     _asyncThenWrapperHelper.<anonymous closure>
      (dart:async-patch/async_patch.dart:73:64)
      #21     _rootRunUnary (dart:async/zone.dart:1134:38)
      #22     _CustomZone.runUnary (dart:async/zone.dart:1031:19)
      #23     _FutureListener.handleValue (dart:async/future_impl.dart:139:18)
      #24     Future._propagateToListeners.handleValueCallback
      (dart:async/future_impl.dart:680:45)
      #25     Future._propagateToListeners (dart:async/future_impl.dart:709:32)
      #26     Future._completeWithValue (dart:async/future_impl.dart:524:5)
      #27     Future._asyncComplete.<anonymous closure>
      (dart:async/future_impl.dart:554:7)
      #28     _rootRun (dart:async/zone.dart:1126:13)
      #29     _CustomZone.run (dart:async/zone.dart:1023:19)
      #30     _CustomZone.runGuarded (dart:async/zone.dart:925:7)
      #31     _CustomZone.bindCallbackGuarded.<anonymous closure>
      (dart:async/zone.dart:965:23)
      #32     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
      #33     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
      #34     _runPendingImmediateCallback
      (dart:isolate-patch/isolate_patch.dart:118:13)
      #35     _RawReceivePortImpl._handleMessage
      (dart:isolate-patch/isolate_patch.dart:175:5)


! Doctor found issues in 1 category.
```

Unplugging the old iPod and rebooting resolved the issue.
